rulesets/default.yml

```diff
@@ -13,8 +13,6 @@ Bundler/GemVersion:
 Layout:
   Exclude:
     - db/schema.rb
-Layout/ArgumentAlignment:
-  EnforcedStyle: with_fixed_indentation
```

Comment:
By deleting this config, we are going back to the default `EnforcedStyle`, which is `with_first_argument`. I forget the exact reason for doing this, but I'm pretty sure that this play better with some of the other cops being added/modified in this change.